server/routes/aws/cloudwatch_tasks.py

```
@@ -0,0 +1,423 @@
+
+def _safe_json(data: Any) -> str:
+    try:
+        return json.dumps(data, ensure_ascii=False)
+    except Exception:
+        return str(data)
+
+
+@celery_app.task(
+    bind=True, max_retries=3, default_retry_delay=30, name="cloudwatch.process_alarm"
+)
+def process_cloudwatch_alarm(
```

Ruff (0.15.14)

[warning] Too many return statements (7 > 6)

(PLR0911)